docs/snippets/logger/unitTesting.ts

```diff
@@ -1,25 +1,25 @@
 const dummyContext = {
```

Comment:
Let's remove this block and import the dummy context from the `commons` package [like we do here](https://github.com/awslabs/aws-lambda-powertools-typescript/blob/main/packages/tracer/tests/unit/Tracer.test.ts#L7).